index.html

```diff
@@ -0,0 +1,151 @@
+ <!DOCTYPE html>
+<html lang="en-gb">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet" />
+	<link href="style.css" type="text/css" rel="stylesheet" />
+	<link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+		<title>Chris Walton</title>
+</head>
+
+<body>
+	<section class="nav-container">
+		<div class="nav-links">
+			<a href="#about-me">About</a>
+			<a href="#portfolio">Portfolio</a>
+			<a href="#contact-me">Contact</a>
+		</div>
+		<div class="nav-cv">
+			<a href="crwaltoncv.docx">Download CV</a>
+		</div>
+		<div class="nav-drop">
+			<button class="dropbtn">Menu</button>
+			<div class="dropdown-content">
+				<a href="#about-me">About</a>
+				<a href="#portfolio">Portfolio</a>
+				<a href="#contact-me">Contact</a>
+				<a href="#crwaltoncv.docx">Download CV</a>
+			</div>
+		</div>	
+	</section>
+	
+	<header>
+		<h1>Chris Walton</h1>
+	</header>
+	
+	<section class="about-container" id="about-me">
+		<div class="about-writing">
+			<div class="about-align">
+				<h3>ABOUT ME</h3>
+				<br>
+				<p>I am a software developer based in Cardiff, with an interest in "INSERT AS APPROPRIATE". I will graduate from iO Academy in October 2022.</p>
+				<br>
+				<p>Prior to my training, I was completing a biology degree and have a deep interest in conservation and the environment. My strengths include communication skills and problem-solving.</p>
+				<br>
+				<p>Outside of work I enjoy reading science fiction novels, playing Dungeons & Dragons with a group of close friends, hiking around South Wales and watching rugby. I also have an minor obsession with cooking and love discovering new restaurants.</p>
+			</div>
+		</div>
+		<div class="gallery">
+			  <img src="me.jpg" alt="A photo of myself" />
+			  <img src="food.jpg" alt="A photo of a dessert" />
+			  <img src="dnd.jpg" alt="Some D&D dice" />
+			  <img src="rugby.jpg" alt="Rugby player setting up a kick" />
+		</div>
+	</section>
+	
+	<section class="port-container" id="portfolio">
+		<div class="port-head">
+			<h3>My Projects</h3>
+			<br> 
+			<p>This section contains a few highlighted projects that I have worked on.<br>If you wish to see more please check out my GitHub.</p>
+		</div>
+		<div class="article-container">
+			<article>
+				<img src="website.png" alt="website image" />
+				<a href="#" class="web-link">Project Name</a>
+				<br>
+				<p>Information about the about project</p>
+				<br>
```

Comment:
You should be using margin or padding for spacing, not br tags. The only time a br tag is allowed is inside of a <p> tag to add lines into a single paragraph - but even then you could argue that it should be multiple paragraphs.

Reply:
This applies to the other items in this section